wiki/Tournaments/o!mAC/2019/en.md

```diff
@@ -0,0 +1,89 @@
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile Badge (To be confirmed), 1 month supporter tag, userpage banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 1 month supporter tag, userpage banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | userpage banner |
+
+## Organisation
+
+The osu!mania Asian's Champions 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_ID] [chouyaa](https://osu.ppy.sh/users/8404646) |
+| Co-host | ![][flag_PH] [vincent4399](https://osu.ppy.sh/users/9764388) |
+| Map Selector | ![][flag_ID] [Virtue-](https://osu.ppy.sh/users/6144772), ![][flag_TH] [RuleBlazing](https://osu.ppy.sh/users/7312402), ![][flag_SG] [riunosk](https://osu.ppy.sh/users/5594381) |
+| Streamer | ![][flag_DE] [DinX](https://osu.ppy.sh/users/568111), ![][flag_NL] [NightNarumi](https://osu.ppy.sh/users/4381142), ![][flag_LV] [Adiopulse](https://osu.ppy.sh/users/4387852) |
+| Commentator | ![][flag_VN] [Weabonie](https://osu.ppy.sh/users/8151986), ![][flag_TH] [HowToPlayLN](https://osu.ppy.sh/users/10879600) |
+| Referee | ![][flag_TH] [TNKSTC](https://osu.ppy.sh/users/6037121), ![][flag_ID] [Noch Einen](https://osu.ppy.sh/users/12668735), ![][flag_ID] [FadillSan](https://osu.ppy.sh/users/7006830), ![][flag_ID] [RiP46](https://osu.ppy.sh/users/5222596), ![][flag_TW] [Makura](https://osu.ppy.sh/users/10665623), ![][flag_FR] [Mathisca](https://osu.ppy.sh/users/2227462) |
+| Statistician | ![][flag_TH] [mashedpotato_old](https://osu.ppy.sh/users/496237) |
+| Wiki | ![][flag_CN] [kj415j45](https://osu.ppy.sh/users/9367540) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/946117)
+- **Statistics sheet**
+  - [Mappool](https://docs.google.com/spreadsheets/d/1aGQHz8YRl-8a8mdp-7E5h_KQBk3rnwlvUzePaIrx6Zs)
+  - [Schedule](https://docs.google.com/spreadsheets/d/1vbSfQXcCG-CMeBmwlCKN5WW1gaaY48yqGsjyFvcN0zU)
+
+## Mappools
+
+### Qualifiers
+
+- [Culprate & Dictate - Pencilina \[Stage 1 : Paranoid\]](https://osu.ppy.sh/beatmapsets/991059#mania/2072838)
+- [Hyper Potions - Ski \[Stage 2: Lubricity\]](https://osu.ppy.sh/beatmapsets/1030043#mania/2153826)
+- [senya - Kimi to Dareka no Yasashisa ni \[Stage 3 : Great Harmony\]](https://osu.ppy.sh/beatmapsets/981019#mania/2053009)
+- [SOUND VOLTEX IV HEAVENLY HAVEN (Laur) - Absolute Domination \[Stage 4 : Devastation\]](https://osu.ppy.sh/beatmapsets/1031176#mania/2156349)
+- [東方紅魔郷 ~ the Embodiment of Scarlet Devil. (PHOENIX Project) - Youma Yakou \[Stage 5: Umbrakinesis\]](https://osu.ppy.sh/beatmapsets/1012614#mania/2119478)
+
+## Ruleset
+
+### Tournament Registration
+
+- You must be in Asia continent countries
```

Comment:
```suggestion
- The flag on your profile must be inside one of Asian continent. For complete list you can visit [following page](https://www.worldometers.info/geography/how-many-countries-in-asia/)
```
I'd prefer to specify with the countries that can participate, you can link it to other sites instead of putting all country in the wiki.